app/src/main/java/remy/pouzet/realestatemanager2/datas/database/dao/EstateDao.java

```diff
@@ -14,21 +14,15 @@
 /**
  * Created by Remy Pouzet on 12/11/2020.
  */
-@Dao
-public interface EstateDao {
+@Dao public interface EstateDao {
```

Comment:
Avec un reformat code, voila ce que ca donne :

```java
@Dao
public interface EstateDao {

    @Query("SELECT * FROM Estate ")
    LiveData<List<Estate>> getAllEstates();

    @Query("SELECT * FROM Estate  WHERE mId = :id")
    LiveData<Estate> getEstate(long id);

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    long createEstate(Estate estate);

    @Update
    int updateEstate(Estate estate);

    @Query("DELETE FROM Estate  WHERE mId = :id")
    int deleteEstate(long id);
}
```

Reply:
Pour le Reformat Code je m'en sert déjà souvent et à chaques commit que je fait mais je crois que mes paramètres de formatage ne sont pas les même que les tiens, sans d'apès ce que j'en vois il y a une différence de paramètrage au niveau de editor>code style>java> wrapping and brace mais je n'arrive pas à obtenir le même résultat que toi (retour à la ligne après une paranthèse fermante des paramètres d'une méthode....

Reply:
Voir commentaire précédent, sinon je le ferait a ta place, pas de soucis